components/emoji/emoji_page.tsx

```diff
@@ -33,6 +35,11 @@ export default class EmojiPage extends React.PureComponent<Props> {
     componentDidMount() {
         this.updateTitle();
         this.props.actions.loadRolesIfNeeded(['system_admin', 'team_admin', 'system_user', 'team_user']);
+        Utils.resetTheme();
+    }
+
+    componentWillUnmount() {
+        Utils.applyTheme(this.props.currentTheme);
```

Comment:
Question : how were you able to find out that this is the problem. and why is the theme not being applied in the component in the first place 🤔

Reply:
Oh I think this is a good approach